### ISSUE TITLE
DOLP 集成请求

### ISSUE THREAD

**Issue by Andrest07** [site098/mysterious#1]
您好，我是 DoLP 的开发者。我喜欢您的图形模块，我可以将其添加为图像包吗？

**Comment by site098** [comment]
No problem
But there are some caveats to this mod that I want you to understand: make sure to turn off the skin tone display option before using it, and turn off the skin tone visual performance after loading the archive
The mod does not currently contain any clothes
If a white image appears, try to spend the day from 0:00 to the next day
Suggested Solution: Use the mod to turn off skin tone visuals by default: don't underestimate the player's behavior of not reading the instructions, which can annoy you to death.
The white image is bug for unknown cause, presumably a loading issue, and I couldn't fix it





------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月29日(星期四) 下午4:03
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [site098/mysterious] DOLP 集成请求 (Issue #1)














 
您好，我是 DoLP 的开发者。我喜欢您的图形模块，我可以将其添加为图像包吗？

**Comment by site098** [comment]
【以下是原文】没问题
但这个模组有一些注意事项希望你了解：使用前确认关闭肤色显示选项,加载存档后再关：选项，性能，关闭肤色视觉表现
该模组目前不包含任何衣服
出现白色图像请尝试度过一天的0：00到第二天
建议解决方案：使用该模组时将关闭肤色视觉表现设为默认：不要低估玩家不看说明的行为，能将你们烦死。
白色图像为未知原因bug，推测是加载问题，我没能解决它





------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月29日(星期四) 下午4:03
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [site098/mysterious] DOLP 集成请求 (Issue #1)














 
您好，我是 DoLP 的开发者。我喜欢您的图形模块，我可以将其添加为图像包吗？

**Comment by Andrest07** [comment]
> No problem But there are some caveats to this mod that I want you to understand: make sure to turn off the skin tone display option before using it, and turn off the skin tone visual performance after loading the archive The mod does not currently contain any clothes If a white image appears, try to spend the day from 0:00 to the next day Suggested Solution: Use the mod to turn off skin tone visuals by default: don't underestimate the player's behavior of not reading the instructions, which can annoy you to death. The white image is bug for unknown cause, presumably a loading issue, and I couldn't fix it
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: ***@***.***&gt;; 发送时间: 2024年8月29日(星期四) 下午4:03 收件人: ***@***.***&gt;; 抄送: ***@***.***&gt;; 主题: [site098/mysterious] DOLP 集成请求 (Issue #1) 您好，我是 DoLP 的开发者。我喜欢您的图形模块，我可以将其添加为图像包吗？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;

Yeah, I noticed the issue as well, not sure what's causing it. Either way, it is no problem, thank you for giving permission!
